gitops/README.md

```diff
@@ -0,0 +1,65 @@
+# Initialize the GitOps environment
+## Install the operators
+The `Red Hat OpenShift Pipelines` and `Red Hat OpenShift GitOps` operators can be installed from the solution 
+derived from the [Janus IDP Demo](https://github.com/redhat-gpte-devopsautomation/janus-idp-bootstrap)
+>This repository contains automation to install the Janus IDP Demo, as well as supporting components
+
+A fork has been created to remove the configuration that excludes Tekton resources from being configured from the 
+ArgoCD applications (see [discussion](https://github.com/argoproj/argo-cd/discussions/8674#discussioncomment-2318554)).
+
+First, install the `Red Hat OpenShift Pipelines` operator:
+```console
+git clone https://github.com/parodos-dev/janus-idp-bootstrap.git
+cd charts/pipelines-operator
+helm upgrade --install orchestrator-pipelines . -f values.yaml -n orchestrator-gitops --create-namespace
+```
+
+Finally install and configure the `Red Hat OpenShift GitOps` operator:
+```console
+git clone https://github.com/parodos-dev/janus-idp-bootstrap.git
+cd charts/gitops-operator
+helm upgrade --install orchestrator-gitops . -f values.yaml -n orchestrator-gitops --create-namespace --set namespaces={orchestrator-gitops}
+```
+
+## Installing docker credentials
+To allow the Tekton resources to push to the registry, we need an account capable to push the image to the registry:
+
+* Create or edit a [Robot account](https://access.redhat.com/documentation/en-us/red_hat_quay/3.3/html/use_red_hat_quay/use-quay-manage-repo) and grant it `Write` permissions to the newly created repository
+* Download `the Docker configuration` file for the robot account and move it under the root folder of this repository (we assume the file name is `orchestrator-auth.json`)
```

Comment:
nit ``s/`the Docker configuration`/the `Docker configuration` ``

Reply:
Where is the Docker configuration file located?
Is `orchestrator-auth.json` the "destination" filename? It seems so reading the following steps but maybe we can be more clear here

Reply:
> Where is the Docker configuration file located? Is `orchestrator-auth.json` the "destination" filename? It seems so reading the following steps but maybe we can be more clear here

It was this described in the following item, let me know if it needs more clarity
* Download the `Docker configuration` file for the robot account and move it under the root folder of this repository (we assume the file name is `orchestrator-auth.json`)

Reply:
you're right, I misread it. Thanks :)